napari/_vispy/visuals/vectors.py

```diff
@@ -1,9 +1,50 @@
-from vispy.scene.visuals import Mesh
+import warnings
 
+from ..utils.gl import get_gl_extensions
 from .clipping_planes_mixin import ClippingPlanesMixin
 
+GLPLUS = 'geometry_shader' in get_gl_extensions()
+# GLPLUS = False
```

Comment:
Comment out this line to test locally.

BTW, It would be nice to have some way to test this on CI as well!

Reply:
And of course the CI is screaming because it does not have gl+ and all the warnings are errors :P